client-side/package-lock.json

```diff
@@ -22,8 +22,12 @@
         "express": "^4.18.2",
```

Comment:
Why are there so many changes in the package-lock.json file? Have you installed some new package or dependency?

Reply:
some new dependancies were installed such react datepicker. Other necessary changes have been committed